### PR TITLE
VZ-7152.  Remove MySQL operator from APIs and VPO code base for release-1.4

### PR DESCRIPTION
**Pull request by michael-cico**
Remove MySQL operator from APIs and VPO code base for release-1.4.

Reverts the following commits, plus some manual edits, in reverse chronological order:

Uninstall fixes
0a473a11e2f13a7b9a6a9ab5fd4fed33e19144d6
4f2d4e5e74313b8d6dca1fee9ff7a841a6db705f

Private registry fixes
011b7c5ae29002fa5e829ac7bf60f7440a37be55
f20958588af7d289a2f148724eed57aefb9f4185

 Partial — namespace mappings (manual revert of constants)
c2ed535b3125b83660bbaec27fd198071c3df265

Fluentd configuration for MySQL operator
cac77ce24b1fd88bd8b3e40bb85d551f813ee439

MySQL operator component introduction
5ba2e93ad1bd585aab651b9add0633a0aedc5a7f

API changes to add MySQL operator
b7e0dd55d0678dd4166dbaec09fb968f4940679d

### CODE REVIEW

File: pkg/constants/constants.go [hunk 1]
```diff
@@ -4,9 +4,8 @@
 package constants
 
 import (
-	"time"
```
Comment (michael-cico) [inline pkg/constants/constants.go:7]: fix imports

File: tests/e2e/config/scripts/create_ocir_repositories.sh [hunk 2]
```diff
@@ -178,7 +178,7 @@ function create_image_repos_from_archives() {
     local is_public="false"
     if [ "$from_repository" == "rancher" ] || [ "$from_image_name" == "verrazzano-platform-operator" ] \
       || [ "$from_image_name" == "fluentd-kubernetes-daemonset" ] || [ "$from_image_name" == "proxyv2" ] \
-      || [ "$from_image_name" == "weblogic-monitoring-exporter" ] && [ -z $OCIR_SCAN_TARGET_ID ]; then
+      || [ "$from_image_name" == "mysql-operator" ] || [ "$from_image_name" == "weblogic-monitoring-exporter" ] && [ -z $OCIR_SCAN_TARGET_ID ]; then
```
Comment (michael-cico) [inline tests/e2e/config/scripts/create_ocir_repositories.sh:181]: need to remove mysql-operator from this

File: platform-operator/controllers/verrazzano/uninstall_component.go [hunk 3]
```diff
@@ -48,23 +51,16 @@ func (r *Reconciler) uninstallComponents(log vzlog.VerrazzanoLogger, cr *install
 		return newRequeueWithDelay(), err
 	}
 
-	var requeue bool
```
Comment (michael-cico) [inline platform-operator/controllers/verrazzano/uninstall_component.go:51]: looks like some unrelated reverts here, manually compare

File: pkg/constants/constants.go (hunk 1 above)
Comment (michael-cico) [reply pkg/constants/constants.go]: fixed

File: tests/e2e/config/scripts/create_ocir_repositories.sh (hunk 2 above)
Comment (michael-cico) [reply tests/e2e/config/scripts/create_ocir_repositories.sh]: Fixed

**Review by mgianatagh — APPROVED** [review]

**Review by agv426 — APPROVED** [review]

**Review by stevshap — APPROVED** [review]

**Review by fredtibbitts — APPROVED** [review]

**Review by pfmackin — APPROVED** [review]

**Review by mgianatagh — APPROVED** [review]

**Review by mgianatagh — APPROVED** [review]

**Review by agv426 — APPROVED** [review]

**Pull request merged**